src/writer/blocks/writerchatreplywithtoolconfig.py

```
@@ -0,0 +1,392 @@
+                        },
+                        "error": {
+                            "name": "Error",
+                            "description": "If the function raises an Exception.",
+                            "style": "error",
+                        },
+                    },
+                },
+            ),
+        )
+
+    def _make_callable(self, tool_name: str, app_id: str, function_name: str):
```

Ruff (0.14.3)

Unused method argument: `tool_name`

(ARG002)